src/main.go

```diff
@@ -140,59 +149,19 @@ func main() {
 	<-interrupt
 }
 
-func startPacketUpdateRoutine(vehicleUpdates <-chan vehicle_models.PacketUpdate, dataTransfer *data_transfer.DataTransfer, loggerHandler *logger_handler.LoggerHandler) {
-	updateFactory := update_factory.NewFactory()
-
-	for packetUpdate := range vehicleUpdates {
-		update := updateFactory.NewUpdate(packetUpdate)
-		dataTransfer.Update(update)
-
-		loggerHandler.Log(packet_logger.ToLoggablePacket(packetUpdate))
-
-		for id, value := range packetUpdate.Values {
-			loggerHandler.Log(value_logger.ToLoggableValue(id, value, packetUpdate.Metadata.Timestamp))
-		}
-	}
-}
-
-func startProtectionsRoutine(vehicleProtections <-chan vehicle_models.ProtectionMessage, messageTransfer *message_transfer.MessageTransfer, loggerHandler *logger_handler.LoggerHandler) {
-	for protection := range vehicleProtections {
-		messageTransfer.SendMessage(protection)
-		loggerHandler.Log(protection_logger.LoggableProtection(protection))
-	}
-}
+func createPid(path string) {
+	err := WritePid(path)
 
-func startOrderRoutine(orderChannel <-chan vehicle_models.Order, vehicle *vehicle.Vehicle, loggerHandler *logger_handler.LoggerHandler) {
-	for ord := range orderChannel {
-		err := vehicle.SendOrder(ord)
-
-		if err != nil {
-			trace.Error().Any("order", ord).Msg("error sending order")
+	if err != nil {
+		switch err {
+		case ErrProcessRunning:
+			trace.Fatal().Err(err).Msg("BACKEND IS ALREADY RUNNING")
```

Comment:
There is no need to capitalize the message
```suggestion
			trace.Fatal().Err(err).Msg("Backend is already running")
```